### PR TITLE
Dbaas 1783: Script to create the splicespark interpreter

### PR DESCRIPTION
**Pull request by edriggers**

### CODE REVIEW

**Review by jramineni — COMMENTED** [review]
I am not sure about the git location of this file. Should it be in splice-zeppelin, where all the install (for cloud) are, may be like standalone install?
Or the standalone does not use any thing from splice-zeppelin, so it belongs here?

**Comment by edriggers** [comment]
This repo 'zeppelin-notebooks' is a public repo and the expectation is that anyone call pull this repo and use this new script to setup splice machine with zeppelin without any cloud specific files.

**Comment by jramineni** [comment]
> This repo 'zeppelin-notebooks' is a public repo and the expectation is that anyone call pull this repo and use this new script to setup splice machine with zeppelin without any cloud specific files.

Ok, then will be good to add setup instructions in the Readme.md or a new Setup.md

**Review by jramineni — APPROVED** [review]

**Comment by edriggers** [comment]
Good Idea - i'll update the README

File: configure-spark-interpreter.py [hunk 1]
```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
```
Comment (MurrayBrown) [inline configure-spark-interpreter.py:1]: will this work on both python 2.7 and 3.x ?  I think we have customers who may use both.

File: configure-spark-interpreter.py [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+                    default='CDH', help='The hadoop vendor.  Valid values are CDH for Cloudera or HDP for Hortonworks')
+
+parser.add_argument('-c', '--hadoopVendorRootDir', dest='hadoopVendorRootDir',
+                    help='The root directory for hadoop.  If the vendor is CDH then it defaults to /opt/cloudera/parcels if it is HDP it defaults to /usr/hdp/2.6.3.0-235')
+
+parser.add_argument('-r', '--resourceUrl', dest='resourceUrl',
+                    default='http://localhost:8088', help='Yarn resource url for example: http://localhost:8088')
+
+parser.add_argument('-s', '--spliceAdditionalJarDir', dest='spliceAdditionalJarDir',
+                    default='/tmp/splicemachine/', help='Path where additional splice machine jars are place')
```
Comment (MurrayBrown) [inline configure-spark-interpreter.py:50]: /tmp is a bad place to permanently store jars. it gets wiped once a month on most unix systems

File: configure-spark-interpreter.py [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+SPLICE_ADDITIONAL_JARS_DIR=args.spliceAdditionalJarDir
+USER=args.user
+PASSWORD=args.password
+ZEPPELIN_VERSION=args.zeppelinVersion
+SPLICE_SPECIAL_JAR=args.spliceSparkInterpreterJarName
+SPLICE_INTERPRETER_NAME = 'splicespark'
+
+if HADOOP_ROOT_DIR is None:
+    if HADOOP_VENDOR == 'CDH':
+        HADOOP_ROOT_DIR = '/opt/cloudera/parcels'
```
Comment (MurrayBrown) [inline configure-spark-interpreter.py:79]: apparently some customers will install CDH to an alternate location. There is cloudera variable 'parcel_dir' that we should try to discover or use.

**Review by MurrayBrown — APPROVED** [review]
I saw a couple places where the script could maybe get 'better', but ... if it works, perhaps ship it as is and file more Jiras for that?

File: configure-spark-interpreter.py (hunk 1 above)
Comment (edriggers) [reply configure-spark-interpreter.py:1]: I only tested with python 2.7

File: configure-spark-interpreter.py (hunk 2 above)
Comment (edriggers) [reply configure-spark-interpreter.py:50]: This is a parameter that can be passed in and set.

File: configure-spark-interpreter.py (hunk 1 above)
Comment (edriggers) [reply configure-spark-interpreter.py:1]: Created jira - https://splicemachine.atlassian.net/browse/DBAAS-1816 to follow up on this.

File: configure-spark-interpreter.py (hunk 3 above)
Comment (edriggers) [reply configure-spark-interpreter.py:79]: Created jira to follow up on this: https://splicemachine.atlassian.net/browse/DBAAS-1816

**Pull request merged**